scrapy/settings/default_settings.py

```diff
@@ -160,6 +160,13 @@
 
 FILES_STORE_S3_ACL = 'private'
 FILES_STORE_GCS_ACL = ''
+FILES_STORAGES = {}
```

Comment:
I don't know if `FILES_STORAGES` and `FILES_STORAGES_BASE` are good names.

Reply:
They seem coherent with similar settings, so :+1: from me.